usr/share/rear/build/default/980_verify_rootfs.sh

```diff
@@ -38,7 +38,8 @@ local broken_binaries=""
 # see https://github.com/rear/rear/pull/1514#discussion_r141031975
 # and for the general issue see https://github.com/rear/rear/issues/1372
 for binary in $( find $ROOTFS_DIR -type f -executable -printf '/%P\n' ) ; do
-    chroot $ROOTFS_DIR /bin/ldd $binary | grep -q 'not found' && broken_binaries="$broken_binaries $binary"
+    # In order to handle relative paths, we 'cd' to the directory containing $binary before running ldd:
+    chroot $ROOTFS_DIR cd $(dirname $binary) && /bin/ldd $binary | grep -q 'not found' && broken_binaries="$broken_binaries $binary"
```

Comment:
The current
<pre>
chroot $ROOTFS_DIR cd $(dirname $binary) && /bin/ldd $binary | ...
</pre>
cannot work because things like "$(dirname $binary) && ..."
need bash in between, both the $(...) and the && need bash.

FYI how to try out such code:
Use in your local.conf
<pre>
KEEP_BUILD_DIR="yes"
</pre>
and then create a ReaR recovery system via "rear mkrescue"
that results on your system the files of the ReaR recovery system
in /tmp/rear.XXXX/rootfs
Then you can test on command line if such commands work,
e.g. on my system:
<pre>
# usr/sbin/rear -d -D mkrescue
...
You should also rm -Rf /tmp/rear.WZtpIU8jAg6v2FK

# find /tmp/rear.WZtpIU8jAg6v2FK/rootfs/ | grep parted
/tmp/rear.WZtpIU8jAg6v2FK/rootfs/bin/parted
/tmp/rear.WZtpIU8jAg6v2FK/rootfs/usr/lib64/libparted.so.2.0.0
/tmp/rear.WZtpIU8jAg6v2FK/rootfs/usr/lib64/libparted.so.2

# mv /tmp/rear.WZtpIU8jAg6v2FK/rootfs/usr/lib64/libparted.so.2.0.0 /tmp/rear.WZtpIU8jAg6v2FK/rootfs/usr/lib64/libparted.so.2.0.0.away

# chroot $ROOTFS_DIR ldd /bin/parted                         
        linux-vdso.so.1 (0x00007ffeb1ba7000)
        libparted.so.2 => not found
        ...

# ROOTFS_DIR="/tmp/rear.WZtpIU8jAg6v2FK/rootfs"
# binary="/bin/parted"

# chroot $ROOTFS_DIR cd $(dirname $binary) && /bin/ldd $binary
chroot: failed to run command 'cd': No such file or directory

# chroot $ROOTFS_DIR cd $(/bin/dirname $binary) && /bin/ldd $binary
-bash: /bin/dirname: No such file or directory
chroot: failed to run command 'cd': No such file or directory

# chroot $ROOTFS_DIR /bin/bash --login -c "cd $( dirname $binary ) && /bin/ldd $binary"
        linux-vdso.so.1 (0x00007ffcdb791000)
        libparted.so.2 => not found
        ...

# chroot $ROOTFS_DIR /bin/bash --login -c "cd $( dirname $binary ) && /bin/ldd $binary" | grep -q 'not found' && broken_binaries="$broken_binaries $binary"

# echo "'$broken_binaries'"
' /bin/parted'
</pre>

Reply:
Aha!  I must have tested two different ways.  Originally, I tested like you did above, and I got the 'cd' error.  That is why my first pull request used 'bash -c'.  Later, I tested by only looking at rear's log.  That didn't show the error, because we are grepping for 'not found' and the cd error doesn't contain that phrase.  At that point, I mistakenly thought 'bash -c' was not necessary.

In the tmp rootfs environment, I get the same results using 'bash --login -c' as with 'bash -c'.  Is '--login' really necessary?  (I've only tested it so far on CentOS 7).

Reply:
Regarding login shell or not cf. my above
https://github.com/rear/rear/pull/1560#issuecomment-342121757
and therein see the link to
https://github.com/rear/rear/issues/862
and therein in particular
https://github.com/rear/rear/issues/862#issuecomment-274068914
and the subsequent comments therein which show
an example where a login shell in between is needed.